### DISCUSSION TITLE
fast5 file unable to download now

### DISCUSSION THREAD

**Discussion by MengZou1** [GoekeLab/sg-nex-data#13] [Data/Download]
Hi I am doing some analysis on RNA modifications and I need to download the raw fast5 file for dRNA-seq data. But the link is not available now. Another question, when your lab will relase other cell line?

**Comment by jonathangoeke** [comment]
Hi @MengZou1 the fast5 files are too large to be hosted with the current file hosting service, but we look into how to make them available, hopefully in the next weeks (likely through ENA). You can sign up to the email list where we will post any update: https://groups.google.com/forum/#!forum/sg-nex-updates/join

**Comment by MengZou1** [comment]
@jonathangoeke Thank you very much. I have joined the group. Hopefully see these raw files in the next weeks.

**Comment by MengZou1** [comment]
Hi @jonathangoeke ,I am wondering I am not recieving the latest news about the raw fast5 data although I have joined the email list. Hoping to see the good news. Thank you.

**Comment by jonathangoeke** [comment]
Hi @MengZou1 yes we still did not finish the upload of the fast5 files, I will post an update here and on the email list then. In the meantime you could use these data here for a start: https://www.ebi.ac.uk/ena/browser/view/PRJEB40872 (from this study: https://www.biorxiv.org/content/10.1101/2020.06.18.160010v1)

**Comment by MengZou1** [comment]
Hi @jonathangoeke , Thank you very much. I will have a careful look.

**Comment by MengZou1** [comment]
Hi @jonathangoeke, Could you tell more information about the data? It is a little difficult  to know which one is raw fast5 for WT form the sample name.

**Comment by jonathangoeke** [comment]
Hi @MengZou1 you can find out from the file names (eg ftp://ftp.sra.ebi.ac.uk/vol1/run/ERR470/ERR4706156/HEK293T-WT-rep1.tar.gz is fast5 for WT; ftp://ftp.sra.ebi.ac.uk/vol1/run/ERR470/ERR4706158/HEK293T-Mettl3-KO-rep1.fastq.gz is fastq for KO). The file format is also described in the additional columns

**Comment by MengZou1** [comment]
Hi @jonathangoeke , Thank you very much and I have found rep1 and rep3 for WT and KO but no rep2?  I am also a little confused about other samples, for example, HEK293T-WT-25-rep4.tar.gz, it has rep4? what is the meaning of "25" ?

**Comment by MengZou1** [comment]
I am wobdering is there something in my download data? Usually it would be stoped for unkown reason. Besides, I could not find the HEK293T-WT-rep2.tar.gz in the list.

**Comment by lwlive** [comment]
I also need the  raw fast5  files, but I can only find fastq files in  EBI(https://www.ebi.ac.uk/ena/browser/view/PRJEB40872)  and  xpore(https://zenodo.org/record/5103099#.YPlv2tOf6UF).  It seems fast5 files are necessary when performing Data preparation from raw reads  in commands "nanopolish index -d <PATH/TO/FAST5_DIR> <PATH/TO/FASTQ_FILE>".
Could you offer me the direct link to donwload the fast5 files? Thank you!

**Comment by ronicaa** [comment]
I'm waiting for the fast5 files as well.

**Comment by jonathangoeke** [comment]
Hi @lwlive @MengZou1 @ronicaa the files from these links are the Hek293T cell line, the fast5 should be available. If the download links are not working, can you post this [here](https://github.com/GoekeLab/xpore/discussions) instead? 

The fast5 files are required for data preprocessing, for analysis with xPore the files from the zenodo link will work.

Thanks!

**Comment by ronicaa** [comment]
@jonathangoeke Thank you very much! I've found those fast5 files!
